tests/parm/model_configure_hafs_shared.IN

```diff
@@ -1,4 +1,3 @@
-print_esmf:              .false.
```

Comment:
Do all applications print esmf by default now?

Reply:
It looks the hafs applications will still not print the logs with change since they're setting 

logKindFlag: ESMF_LOGKIND_NONE

Reply:
ESMF logging control is now handled within nems.configure.

Reply:
HAFS tests do not output the PET log files, so we keep them as they are is by setting the " logKindFlag:            ESMF_LOGKIND_NONE" in  tests/parm/nems.configure.hafs*.IN.

Reply:
That is correct for now. However, my recommendation would be to set **ESMF_LOGKIND_MULTI_ON_ERROR**. It is like ESMF_LOGKIND_NONE if there are no errors. When an error is found, it will switch to ESMF_LOGKIND_MULTI for logging, before the abort.

Reply:
I think all test should use the same log kind. Is there a reason why HAFS test use **ESMF_LOGKIND_MULTI_ON_ERROR** while all other tests use **ESMF_LOGKIND_MULTI**?

Reply:
I like Gerhard's suggestion to set everything to output a log on error. Would it clean up the run directories--that is, does a file (which is essentially empty if no errors) still get created for each PE? Or do we get no PET*log files at all unless there is an error?

Reply:
I think it depends on the test owners. Some test owners use their RT tests for debug purpose and use small number of tasks, while other may use the same nems.configure for their parallel runs which use a large number of tasks, outputting all those PET files may not ideal for them.

Reply:
@DeniseWorthen with **ESMF_LOGKIND_MULTI_ON_ERROR** you get NO PET* files, unless an error is encountered.

Reply:
@DeniseWorthen Gerhard's suggestion is added to the code. Please see hafs nems.configure below. From my previous test there are no pet files when the run finishes successfully.

Reply:
@junwang-noaa, @DusanJovic-NOAA, @DeniseWorthen, @theurich, @MinsukJi-NOAA, As for the EMSF log for all the HAFS related RTs, please feel free to unify and set it the same as other applications. Either ESMF_LOGKIND_MULTI_ON_ERROR or ESMF_LOGKIND_MULTI should be fine for HAFS. Thanks!